### ISSUE TITLE
Availablity of Ubuntu24.04 images

### ISSUE THREAD

**Issue by willcl-ark** [runs-on/runner-images-for-aws#3]
Ubuntu 24.04 images are available on GH runners: https://github.com/actions/runner-images?tab=readme-ov-file#available-images

...but I don't know how to select them in `runs-on`. Are they available to select somehow?

Thanks in advance :)

**Comment by crohr** [comment]
Not yet, but they will be soon!

**Comment by aledbf** [comment]
@crohr is there any public beta image available? (I need cgroups v2 enabled by default)
Thanks

**Comment by crohr** [comment]
@aledbf @willcl-ark hello, ubuntu24 images are now available. They are not yet declared as official images in RunsOn, so for now you would first have to declare them as custom images in your `.github/runs-on.yml` config file:

```yaml
images:
  ubuntu24-beta-x64:
    platform: "linux"
    arch: "x64"
    owner: "135269210855"
    name: "runs-on-v2.2-ubuntu24-full-x64-*"
  ubuntu24-beta-arm64:
    platform: "linux"
    arch: "arm64"
    owner: "135269210855"
    name: "runs-on-v2.2-ubuntu24-full-arm64-*"
```

(I'm using `beta` instead of `full` since I don't want those to conflict with the official image names when they'll be integrated in RunsOn).

And then in your workflows:

```yaml
jobs:
  test:
    runs-on:
      - runs-on
      - runner=2cpu-linux-x64
      - image=ubuntu24-beta-x64
      - run-id=${{ github.run_id }}
```

**Comment by crohr** [comment]
Images are now available when you install the v2.5.4 pre-release of RunsOn (https://github.com/runs-on/runs-on/releases/tag/v2.5.4).
